server/locales/cy.json

```diff
@@ -95,82 +95,39 @@
       "pm": "pm"
     }
   },
-  "settings": {
```

Comment:
This would be much easier to audit / manage if the keys were in the same order in both languages.

Reply:
I think they should already be from what I can see? 🤔

Reply:
Ah the transactions ones weren't sorted! Should be now!